flink-doris-connector/src/test/java/org/apache/doris/flink/sink/writer/TestJsonDebeziumSchemaSerializer.java

```diff
@@ -167,21 +167,23 @@ public void testExtractDDL() throws IOException {
         JsonNode recordRoot = objectMapper.readTree(record);
+        serializer.setOriginFieldSchemaMap(originFiledSchemaMap);
```

Comment:
Is it possible to insert a piece of data at the initial judgment as long as newschema is turned on?

Reply:
Inserting a piece of data actually initializes the originFiledSchemaMap